prometheus/indico_prometheus/controllers.py

```diff
@@ -0,0 +1,55 @@
+# This file is part of the Indico plugins.
+# Copyright (C) 2002 - 2023 CERN
+#
+# The Indico plugins are free software; you can redistribute
+# them and/or modify them under the terms of the MIT License;
+# see the LICENSE file for more details.
+
+from flask import make_response, request
+from flask_pluginengine import current_plugin
+from prometheus_client.exposition import _bake_output
+from prometheus_client.registry import REGISTRY
+from werkzeug.exceptions import Unauthorized
+
+from indico.core.cache import make_scoped_cache
+from indico.web.rh import RH
+
+from indico_prometheus.metrics import update_metrics
+
+
+cache = make_scoped_cache('prometheus_metrics')
+
+
+class RHMetrics(RH):
+    CSRF_ENABLED = False
```

Comment:
Not strictly needed if the endpoint is only using GET (which is probably the case since the url route only allows GET)